tests/test_flow.py

```diff
@@ -3440,7 +3505,10 @@ def test_csv_source_with_none_values():
         "2021-04-21 15:56:53.385444",
     ]
     assert termination_result[1].key == "b"
-    assert termination_result[1].body == ["b", True, None, math.nan, math.nan, None]
+    assert all(
```

Comment:
Need to break down into multiple asserts, or we won't know what actually failed.